GiniVision/Classes/ReviewContainerViewController.swift

```diff
@@ -78,10 +77,54 @@ internal class ReviewContainerViewController: UIViewController, ContainerViewCon
         displayContent(contentController)
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        setupLeftBarButton()
```

Comment:
If you put this in `viewDidLoad`, I guess you would be able to skip the `navigationItem.leftBarButtonItem == nil` in the method.

Is there a reason you put it in `viewWillAppear`?

Reply:
Yes, `navigationController` is still `nil` in `viewDidLoad`.